File: MPI/mpicheck/tests/functions/3.0/MPI_Get_library_version.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    char *var_0;
    int *var_1;
    int ret;
    /* calls */
    ret = MPI_Get_library_version(var_0, var_1);
    ret = PMPI_Get_library_version(var_0, var_1);
    return 0;
}

File: MPI/mpicheck/tests/functions/3.0/MPI_Get_library_version.f90

        
        program main
        use mpi
        
        CHARACTER*(10) var_0
       INTEGER var_1
       INTEGER var_2
        call mpi_get_library_version(var_0, var_1, var_2)
       call pmpi_get_library_version(var_0, var_1, var_2)
        end program main
    